pandas/core/categorical.py

```diff
@@ -179,13 +191,13 @@ class Categorical(PandasObject):
     [a, b, c, a, b, c]
     Categories (3, object): [a < b < c]
 
-    >>> a = Categorical(['a','b','c','a','b','c'], ['c', 'b', 'a'], ordered=True)
+    >>> a = Categorical(['a','b','c','a','b','c'], ['c', 'b', 'a'], \
+                ordered=True)
```

Comment:
Examples shouldn't use `\` for continuation. This line should just be formatted as you would do with normal python code, so something like 

```
>>> a = Categorical(['a','b','c','a','b','c'], ['c', 'b', 'a'],
                    ordered=True)
```

There are also some cases of this in other examples I think